settings/pyrefly.vim

```diff
@@ -0,0 +1,14 @@
+augroup vim_lsp_settings_pyrefly
+  au!
+  LspRegisterServer {
+      \ 'name': 'pyrefly',
+      \ 'cmd': {server_info->lsp_settings#get('pyrefly', 'cmd', [lsp_settings#exec_path('pyrefly')]+lsp_settings#get('pyrefly', 'args', ['lsp']))},
+      \ 'root_uri':{server_info->lsp_settings#get('pyrefly', 'root_uri', lsp_settings#root_uri('pyrefly'))},
+      \ 'initialization_options': lsp_settings#get('pyrefly', 'initialization_options', v:null),
+      \ 'allowlist': lsp_settings#get('pyrefly', 'allowlist', ['python']),
+      \ 'blocklist': lsp_settings#get('pyrefly', 'blocklist', []),
+      \ 'config': lsp_settings#get('pyrefly', 'config', lsp_settings#server_config('pyrefly')),
+      \ 'workspace_config': lsp_settings#get('pyrefly', 'workspace_config', {}),
```

Comment:
If user want to show type error, user should set `python.pyrefly.displayTypeErrors: force-on`

default:

<img width="730" height="365" alt="Image" src="https://github.com/user-attachments/assets/4b020b19-6d80-4415-9c13-c99cd8ce648b" />

force-on:

<img width="730" height="370" alt="Image" src="https://github.com/user-attachments/assets/57a606de-8833-494f-bc09-b57956ec7da5" />